README.rst

```diff
@@ -65,10 +65,10 @@ so using it with libraries like gevent is out of the question, and its
 dependency on libmemcached poses challenges (e.g., it must be built against
 the same version of libmemcached that it will use at runtime).
 
-Python-memcache
----------------
+Python-memcached
```

Comment:
For consistency, let's also spell this as:

```suggestion
python-memcached
```

Reply:
Done, thanks for the suggestion.